src/main/java/training/endava/skeleton/db/Main.java

```diff
@@ -10,15 +15,82 @@ public static void main(String Args[])
         Person test1 = new Person(2, "Diana", "0746", "Vaslui");
         Person test2 = new Person(3, "Alina", "0747", "Suceava");
         Person test3 = new Person(4, "Ada", "0744", "Botosani");
+        Person test4 = new Person(3, "Alina", "0747", "Suceava");
+        Person test5 = new Person(4, "ASfas", "074412", "Iasi");
 
+        //day1 exercises
         ArrayList<Person> mylist = new ArrayList<Person>();
 
-        mylist.add(test0);
-        mylist.add(test1);
-        mylist.add(test2);
-        mylist.add(test3);
+        List<Company> companyList = new ArrayList<>();
+        List<Person> personList1 = new ArrayList<>();
+
+        //Company conti
+        Company comp1 = new Company("Conti",1999,personList1,Company.compArea.IT);
+        personList1.add(test0);
+        personList1.add(test1);
+        personList1.add(test2);
+
+        companyList.add(comp1);
+
+        //Company  Tast
+        List<Person> personList2 = new ArrayList<>();
+        Company comp2 = new Company("Tast",2000,personList2,Company.compArea.FINANCE);
+        personList2.add(test3);
+        personList2.add(test4);
+        personList2.add(test5);
+
+        companyList.add(comp2);
+
+        List<String> companyesNameList = companyList.stream().map(Company::getName).collect(Collectors.toList());
+        System.out.println(companyesNameList);
+
+        int [] companySortYear =companyList.stream().
+                map(Company::getfYear)
+                .mapToInt(Integer::intValue).sorted().toArray();
+
+        for(int i = 0; i<companySortYear.length;i++)
+            System.out.println(companySortYear[i]);
+        //ex 3
+        Company oldestComp = companyList.stream().
+                min((company1,company2)->Integer.compare(company1.getfYear(),company2.getfYear())).get();
+
+        //ex 4
+        Integer countPersons = companyList.stream().
+                map(Company::getListOfPers).
+                map(List::size).
+                reduce(0,(e1,e2)->e1+e2).intValue();
+        System.out.println(countPersons);
+
+        //6
+        companyList.stream().map(a -> a.getListOfPers()).flatMap(List<Person>::stream).
```

Comment:
1. There is no need to specify the class `<Person>`.
2. It is preferred to use method reference instead. Replace lambdas everywhere is applicable.